packages/sql-openapi/lib/entity-to-routes.js

```diff
@@ -185,63 +196,76 @@ async function entityPlugin (app, opts) {
     const targetColumnCamelcase = camelcase(relation.column_name)
     // In this case, we navigate the relationship so we MUST use the column_name otherwise we will fail in case of recursive relationships
     // (or multiple relationships between the same entities). We might want to specify this in documentation, because can be confusing
-    const targetRelation = relation.column_name.replace(/_id$/, '')
+    let targetRelation = relation.column_name.replace(/_id$/, '')
+    if (mapRoutePathNamesRelations.get(targetRelation)) {
+      idxRoutePathNamesRelations++
```

Comment:
Could you please use the property name instead?

Reply:
Nice suggestion! 👍🏼 Added on edad13f

Reply:
I had to revert the changes because the `constraint_name` in some DB engines is `undefined`.

This situation shouldn't happen much. Let's say that it is more to be compatible with an existing DB than an actual feature: having the exact same foreign keys, on the same table, pointing to the exact same primary key, and just renaming the FKs, doesn't make a lot of sense. So it's more of a "safeguard" to avoid Platformatic crashing.